src/utils/generators.mjs

```diff
@@ -53,3 +53,43 @@ export const coerceSemVer = version => {
 
   return coercedVersion;
 };
+
+/**
+ * Gets compatible versions for an entry
+ *
+ * @param {string | import('semver').SemVer} introduced
+ * @param {Array<Release>} releases
```

Comment:
It may be useful to add the `Release` object used here as a type, since it is not currently defined; everything under this object currently appears as `any` 🙇